src/sonic-yang-models/tests/yang_model_tests/tests/crm.json

```diff
@@ -0,0 +1,86 @@
+{
+    "CRM_WITH_CORRECT_FREE_VALUE": {
```

Comment:
Say we have one more field
{
'test_config_file': 'crm.json'
}
This will be parsed as non test and will help to create self.testFile dict.

Reply:
Total time for the test config load
```
INFO:YANG-TEST:Time to load config:0:00:00.002771
```